### ISSUE TITLE
Transition IRC channel to Libera Chat

### ISSUE THREAD

**Issue by yaakov-h** [SteamRE/SteamKit#989]
It's a bit of a shame to do this so soon after #846, but oh well.

Given the recent Freenode drama, the 25-year-old network seems to be somewhere between "volatile" and "hostile" with new staff taking over channels, retroactively changing policies, and so on.

For Libera we should [register a channel](https://libera.chat/chanreg#how-to-register-as-a-project) which gives us control of the channel, control of the "namespace", custom cloaks, and protection against somebody else coming along and trying to register the same channel, e.g. a name collision (unlikely) or maliciously trying to take over.

I've had a brief chat with one of the staff about registration already, so the next steps would be an email projects@libera.chat to apply. In the meantime, we can all leave the channel and first to rejoin should get operator status, but we dont get the benefits listed above.

Outstanding questions:
1. What name do we want? Should we keep it as #steamkit or should we perhaps be #steamre?
2. Which users should be the "Group Contacts" who are effectively admins/contacts for the community (can register channels and request cloaks)?
I think that we should have two at minimum in case somebody disappears or is offline for an extended period, and I don't have any thoughts on a maximum.
3. Do we want it to be set up so that only "Group Contacts" can register channels in our namespace? I don't think this is strictly necessary for us but it might be nice to enable anyway.
4. Do we want to be on the public Libera community list?

@xPaw: can Wendy be reconfigured to post to Libera, or is Wendy going to be Discord-only now that SteamDB has moved in a different direction?

If you want to be a Group Contact please comment here with your NickServ-registered nickname on Libera.

cc/ @VoiDeD @asherkin @psychonic

**Comment by xPaw** [comment]
I shut down my irc stuff. For Lounge, we asked jess (libera staffer) to setup BitBot for GitHub notifications, perhaps you can do the same.

**Comment by VoiDeD** [comment]
1. Either sounds fine to me - but I imagine if we go #steamre then we can get other namespaced channels in the future like #steamre-steamkit, #steamre-depotdownloader, etc. Depends on how much things take off I suppose.
2. I would say all the active/willing team members of our github org.
3. I _think_ so? Seems odd to let anybody register channels that could be perceived to represent us.
4. Sure.

My nick is VoiDeD.

cc @azuisleet as well for all this.

**Comment by yaakov-h** [comment]
Oops, I thought I had tagged Azu.

I've submitted an initial registration request, hopefully they'll get back to us soon but I expect they're under some massive load right now.

We have a ticket number: \[Libera.Chat #&#8203;690\]

**Comment by yaakov-h** [comment]
Freenode are doing dodgy shit again so I'm calling it.

I've updated the topics in both locations and merged #991. We're still waiting for Libera to officially register us as a community but there should be no harm in continuing our semi-registered state where we at least have ChanServ.

I've also reached out in #bitbot to ask about using their hosted bot rather than setting up our own.

I'll leave this open until everything is completed.
